server/src/test/java/org/kaazing/gateway/server/context/resolve/AcceptOptionsTest.java

```diff
@@ -74,13 +77,13 @@ public void setAllowedServices() {
         serviceList.add("directory");
     }
 
-    @Test
+    @Test @Ignore
```

Comment:
Do you want to ignore these tests in this file ? or just forgot to remove the annotation ?